src/lambda/models/samLambdaRuntime.ts

```diff
@@ -56,10 +61,10 @@ const cloud9SupportedCreateRuntimes = cloud9SupportedBaseRuntimes.filter(
 export const samLambdaImportableRuntimes: ImmutableSet<Runtime> = ImmutableSet.union([nodeJsRuntimes, pythonRuntimes])
 
 export const samLambdaCreatableRuntimes: ImmutableSet<Runtime> = isCloud9()
-        ? cloud9SupportedCreateRuntimes
-        // Filter out node8 until local debugging is no longer supported, and
-        // it can be removed from samLambdaRuntimes
-        : samZipLambdaRuntimes.filter((runtime: string) => runtime !== 'nodejs8.10')
+    ? cloud9SupportedCreateRuntimes
+    : // Filter out node8 until local debugging is no longer supported, and
+      // it can be removed from samLambdaRuntimes
+      samZipLambdaRuntimes.filter((runtime: string) => runtime !== 'nodejs8.10')
```

Comment:
Since you are editing this line, node 8 can 100% be removed. We removed it from JB quite a while ago

Reply:
non-trivial and breaking, will do so in a followup PR which will follow this release.